dev/how-to/deploy/tidb-in-kubernetes-prerequisites.md

```diff
@@ -0,0 +1,141 @@
+
+## Kubernetes requirements for system resources
+
+- Each machine requires a relatively large SAS disk (at least 1T) to store the data directories of Docker and kubelet.
+
+  > **Note:**
+  >
+  > The data from Docker mainly includes image and container logs. The data from kubelet are mainly data used in [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir).
+
+- To deploy a monitoring system for the Kubernetes cluster whose data is to be stored on disk, prepare a large SAS disk for Prometheus and the log monitoring system. It is recommended to prepare two large SAS machines for each machine.
+
+  > **Note:**
+  >
+  > In a production environment, it is recommended to use RAID 5 for these disks for the above two usages.
+
+- It is recommended that the number of etcd be consistent with the Kubernetes master nodes. The etcd data is recommended to be stored on the SSD disk.
+
+## TiDB cluster's requirements for resources
+
+The TiDB cluster consists of three components: PD, TiKV and TiDB. The following recommendations on capacity planning is based on the standard TiDB cluster, namely three PDs, three TiKVs and two TiDBs:
+
+- PD component: 2C 4GB. PD occupies relatively less resources and only a small portion of local disks are used.
+
+  > **Note:**
+  >
+  > For easier management, you can put the PDs of all clusters on the master node. For example, to support 5 TiDB clusters, you can plan to deploy 5 PD instances on each of the 3 master nodes. These PD instances use the same SSD disk (disk of 200 or 300 GigaBytes) on which you can create 5 directories as a mount points by means of bind mount. For detailed operation, check [documentation](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md#sharing-a-disk-filesystem-by-multiple-filesystem-pvs) for reference.
+  >
+  > If more machines are added to support more TiDB clusters, you can continue to add PD instances in this way on the master. If the resources on the master are exhausted, you can find other worker nodes to add PDs in the same way. This method is convenient for the planning and management of PD instances. However, if two machines go down, all TiDB clusters will be unavailable due to the concentration of PD instances.
+  >
+  > Therefore, it is recommended to take out an SSD disk from each machines in the cluster to provide PD instances like the master node. If you need to enlarge capacity by adding machines in some clusters, you only need to create PD instances on the newly added machines.
+
+- TiKV components: An NVMe disk of 8C 32GB for each TiKV instance. To deploy multiple TiKV instances on one machine, you must reserve enough buffers when planning capacity.
+
+- TiDB component: 8C 32 GB. Because TiDB component does not occupy the disk, you only need to consider the CPU and memory resources for it in your plan. The following example assumes that the capacity is 8C 32 GB.
```

Comment:
```suggestion
- TiDB component: 8C 32 GB capacity. Because the TiDB component does not occupy the disk space, you only need to consider the CPU and memory resources when planning. The following example assumes that the capacity is 8C 32 GB.
```